src/styles/utils/_mixins.scss

```diff
@@ -4,3 +4,65 @@
     #{$_property}: $_toValue;
   }
 }
+
+@mixin pageGrid {
```

Comment:
```suggestion
@mixin page-grid {
```
Consider using kebab-case for naming 
<img width="544" alt="Screenshot 2024-04-15 at 17 02 16" src="https://github.com/mate-academy/Kickstarter/assets/135232001/4d419fc5-f248-408e-af31-b4779f8a4987">